lib/pages/chat/events/message/message.dart

```diff
@@ -65,6 +68,8 @@ class Message extends StatelessWidget {
   /// of touchscreen.
   static bool useMouse = false;
 
+  static const int maxCharactersDisplayName = 68;
```

Comment:
put it in app config since you are using it multiple times

Reply:
Currently , we only use it here

Reply:
Updated